mapiproxy/servers/default/asyncemsmdb/dcesrv_asyncemsmdb.c

```diff
@@ -183,6 +183,9 @@ static enum mapistore_error build_mapistore_sogo_url(TALLOC_CTX *_mem_ctx, char
 	TALLOC_CTX	*mem_ctx;
 	char		*url_tmp = NULL;
 	char		*_uri = NULL;
+	char		*_username = NULL;
```

Comment:
IMHO, this name is quite misleading, I'd suggest to use `encoded_username` or something.